public/app/features/explore/state/main.ts

```diff
@@ -168,6 +169,8 @@ export const initialExploreState: ExploreState = {
   largerExploreId: undefined,
   maxedExploreId: undefined,
   evenSplitPanes: true,
+  showQueryHistory: false,
```

Comment:
`showQueryHistory` doesn't need to be saved in Redux, could be a simple `useState(...)`